services/initService.ts

```
@@ -93,8 +93,7 @@ const getDefaultSectionIdFromUser = async (todoistToken: string): Promise<string
         updateConfigValue("sections", sections)
         return sectionId;
     } catch (error) {
-        console.error(chalk.red("Error fetching projects:"), error);
-        process.exit(1)
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)